service.unmanic/contextitem.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
```

Comment:
Please use SPDX identifier in Python files (so this goes for all files).

Reply:
done

Reply:
Still missing the SPDX identifier instead of the full copyright part.